Cartfile.resolved

```diff
@@ -1,2 +1,2 @@
-github "AliSoftware/OHHTTPStubs" "8.0.0"
-github "venmo/DVR" "v2.0.1"
+github "vkalta/DVR" "8bb79349473d1ff595f07d216662ee5163854f42"
+github "vkalta/contentstack-utils-swift" "dacaf16bd8e4e182e0bef57df2957c7c8697082c"
```

Comment:
@vkalta Please update the repo name and ensure no secrets are included in the PR.

Reply:
This is not a secret @sunil-lakshman, this is commit hash to ensure the carthage installs from that specific commit.
The repos used are updated with the target platform version which is iOS 12. Previously it was 11 and was causing the build to break.